app/forms/custom_fields/hierarchy/item_form.rb

```diff
@@ -46,16 +46,30 @@ class ItemForm < ApplicationForm
             validation_message: validation_message_for(:label)
           )
 
-          input_group.text_field(
-            name: :short,
-            label: "Short",
-            value: @target_item.short,
-            visually_hide_label: true,
-            full_width: false,
-            required: false,
-            placeholder: I18n.t("custom_fields.admin.items.placeholder.short"),
-            validation_message: validation_message_for(:short)
-          )
+          if @input_format == :short
+            input_group.text_field(
+              name: :short,
+              label: "Short",
```

Comment:
:red_circle: I18n seems to be missing here and below for the score.

edit: And apparently above for the "Label" as well... I can confirm this locally. The thing is that the Label is only rendered for screenreaders, but that doesn't make it less important.